onnx/reference/ops/op_regex_full_match.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) ONNX Project Contributors
+
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=R0912,R0913,W0221
+
+import numpy as np
+import re2
+
+from onnx.reference.op_run import OpRun
+
+_acceptable_str_dtypes = ("U", "O")
+
+
+class RegexFullMatch(OpRun):
+    def _run(self, x, pattern):
```

Comment:
Replace by `pattern=None` otherwise the reference implementation considers it as an input.